### ISSUE TITLE
How to modify injectable JavaScript on server side in React Native?

### ISSUE THREAD

**Issue by singhlovey** [alinz/react-native-webview-bridge#261]
Hi 

I have open simple page with one form using webView component. Now what I want is whenever it gets submit, I want success message under my React Native code. In webView we can inject JavaScript from our React Native code to server side and it works. But I want to modify that JavaScript on server side and want to pass dynamic success message which I can use under my React Native code.

If there is any other way, do let me know as well.

Please help!!

Thanks

**Comment by simonso-nzme** [comment]
I am getting similar issue. Works on Android but not iOS, injectJS works but not server side. When I log the try and catch error, I got: ReferenceError: Can't find variable WebViewBridge

react-native 0.57.1
react-native-webview-bridge 0.40.1

**Comment by simonso-nzme** [comment]
Ok, we got it going at the end. I believe it is a timing/initiation issue. What we end up doing is:

React Native:
`
<WebViewBridge
  injectedJavaScript="window.configureWebView(WebViewBridge)"
/>
`

Web:
`
window.configureWebView = (vBridge) => {
  vBridge.onMessage = (data) => { ... }
}
`

**Comment by hkanizawa** [comment]
@singhlovey I'm with the same problem here.
Did you solve this?

**Comment by djsupersonicdesign** [comment]
Hi anyone solve this?

**Comment by fungilation** [comment]
No, this repo is dead to me. I've migrated completely to https://github.com/react-native-community/react-native-webview which is the new official `webview` of RN, it's actively developed and it does everything that `react-native-webview-bridge` did. I suggest you to migrate as well.
